<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GatewayBasicTabWidget</class>
 <widget class="QTabWidget" name="GatewayBasicTabWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gateway</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Basic</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QGroupBox" name="basicGroup">
      <property name="title">
       <string>Basic</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Baudrate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="periodSpin">
           <property name="suffix">
            <string>(ms)</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>UPDATE PIRIOD</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="baudCombo">
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="displayFormat">
            <string>yyyy/M/d H:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="hostDatetimeBtn">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="addressEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QPushButton" name="basicUploadBtn">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="basicGetFromDbBtn">
        <property name="text">
         <string>READ FROM DB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="basicSaveToDbBtn">
        <property name="text">
         <string>WRITE TO DB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="basicDownloadBtn">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="ResetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="networkGroup">
      <property name="title">
       <string> Network</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="ethernetGroup">
           <property name="title">
            <string>Ethernet</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>ip</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="gatewayEdit">
              <property name="text">
               <string>192.168.111.1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="portEdit">
              <property name="text">
               <string>502</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>port</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>gateway</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="maskEdit">
              <property name="text">
               <string>255.255.255.0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ipEdit">
              <property name="text">
               <string>192.168.111.1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>mask</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="serial">
           <property name="title">
            <string>RS485</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>parity</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="rs485BaudCombo">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>1200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4800</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>57600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>stopbits</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="rs485StopbitsCombo">
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>baudrate</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="rs485DatabitsCombo">
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>databits</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="rs485ParityCombo">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Even</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Odd</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>Schedule</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="getScheduleCountInDbBtn">
      <property name="text">
       <string>Schedule Count(DB)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="scheduleCountInDbEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="7">
     <widget class="QGroupBox" name="verticalGroupBox2">
      <property name="title">
       <string>Day</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignRight">
        <widget class="QListWidget" name="dayList"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="getScheduleCountInDevBtn">
      <property name="text">
       <string>Schedule Count(DEV)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="scheduleCountInDevEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Schedule Order No.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="scheduleOrderSpin">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="scheduleEnableCheck">
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Trigger Time</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="title">
       <string>SceneList</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableWidget" name="sceneTable"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1" rowspan="2">
     <widget class="QGroupBox" name="verticalGroupBox_2">
      <property name="title">
       <string>Month</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="monthList"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QGroupBox" name="verticalGroupBox1">
      <property name="title">
       <string>Weekday</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="weekdayList"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="advUploadBtn">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="advDownloadBtn">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="advGetFromDbBtn">
        <property name="text">
         <string>READ FROM DB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="advSaveToDbBtn">
        <property name="text">
         <string>WRITE TO DB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
